superset/views/base.py

```diff
@@ -166,6 +167,9 @@ def api(f: Callable[..., FlaskResponse]) -> Callable[..., FlaskResponse]:
     def wraps(self: "BaseSupersetView", *args: Any, **kwargs: Any) -> FlaskResponse:
         try:
             return f(self, *args, **kwargs)
+        except NoAuthorizationError as ex: # pylint: disable=broad-except
+            logger.exception(ex)            
```

Comment:
We probably don't want to log this as an exception, since it's not a server error.

Reply:
+1, lets just log as a warning since this isn't really actionable